app/views/admin/page/index.html.erb

```diff
@@ -1,17 +1,24 @@
 <div class="govuk-grid-row">
 
-  <div class="govuk-grid-column-two-thirds">
-    &nbsp;
-  </div>
+  <div class="govuk-grid-column-one-half">
+    <h2 class="govuk-heading-m">Claims awaiting checking</h2>
+    <p class="govuk-body govuk-!-font-size-36"><%= @claims_awaiting_checking.count %></p>
+    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
 
-  <div class="govuk-grid-column-one-third">
     <h2 class="govuk-heading-m">Total claims received</h2>
     <p class="govuk-body govuk-!-font-size-36"><%= Claim.submitted.count %></p>
-
     <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+  </div>
 
-    <h2 class="govuk-heading-m">Claims awaiting checking</h2>
-    <p class="govuk-body govuk-!-font-size-36"><%= Claim.awaiting_checking.count %></p>
+  <div class="govuk-grid-column-one-half">
+    <h2 class="govuk-heading-m">Claims approaching deadline</h2>
+    <p class="govuk-body govuk-!-font-size-36"><%= @claims_awaiting_checking.select(&:approaching_deadline?).count %></p>
```

Comment:
This is going to load all the claims into memory to perform the count. Is there a way to calculate this number, and the number below using a query instead and set them as instance variables in the controller action?